gdal/src/test/scala/geotrellis/contrib/vlm/spark/GDALRasterSummarySpec.scala

```diff
@@ -42,7 +42,7 @@ class GDALRasterSummarySpec extends FunSpec with TestEnvironment with BetterRast
           .map(uri => GDALRasterSource(uri): RasterSource)
           .cache()
 
-      val summary = RasterSummary.fromRDD[RasterSource, Long](sourceRDD)
+      val summary = RasterSummary.fromRDD(sourceRDD)
```

Comment:
Even though I added an extra type parameter into the `RasterSummary`, it is still possible to simplify signatures.